woocommerce/wc-admin
# Revert WCPay international support for bundled package

Reverts the international support for WCPay due to incomplete agreements.

Relevenat thread - paJDYF-15E-p2

cc @pmcpinto to confirm that this is the only flag we need to turn off.

### Detailed test instructions:

1. Build the core version of this branch.
2. Set your store to one of the following countries: `'AU', 'CA', 'GB', 'IE', 'NZ'`.
3. Make sure WCPay is not shown as a recommendation in the task list or payments task.
